src/Microsoft.CodeQuality.Analyzers/UnitTests/Maintainability/AvoidDeadConditionalCode_NullAnalysis.cs

```diff
@@ -4780,20 +4811,20 @@ Class Test(Of T As C)
             Return
         End If
 
-        Dim d = DirectCast(t, D)    '  Compiler error BC30311: Value of type 'T' cannot be converted to 'D'
+        Dim d = DirectCast({|BC30311:t|}, D)
```

Comment:
💡 It would be good to keep the comment since the method name doesn't clearly indicate that the test covers compiler error cases.

```suggestion
        Dim d = DirectCast({|BC30311:t|}, D)    '  Compiler error BC30311: Value of type 'T' cannot be converted to 'D'
```